README.md

```diff
@@ -135,7 +135,7 @@ class {'satellite_pe_tools':
 }
 ~~~
 
-This example tells the master the location of the Satellite server (`https://puppet.example.com`) and instructs it to verify the Satellite server's identity.
+This example tells the server the location of the Satellite server (`https://puppet.example.com`) and instructs it to verify the Satellite server's identity.
```

Comment:
Server is an ambiguous name, which is my whole problem with this naming.
```suggestion
This example tells the Puppet server the location of the Satellite server (`https://puppet.example.com`) and instructs it to verify the Satellite server's identity.
```

Reply:
Have added, the Puppet bit, you have a god point, with the two servers on the one line it can get confusing